### PR TITLE
Version 285

### PR DESCRIPTION
**Pull request by madmongo1**

### CODE REVIEW

File: include/boost/beast/http/impl/file_body_win32.hpp [hunk 1]
```diff
@@ -337,6 +337,46 @@ class null_lambda
     }
 };
 
+// https://github.com/boostorg/beast/issues/1815
+// This function mimics the behaviour of ASIO.
```
Comment (vinniefalco) [inline include/boost/beast/http/impl/file_body_win32.hpp]: I would write
```
// RHODGES This function mimics the behaviour of ASIO....
```
To make it clear that this is developer commentary and not documentation about the code

Comment (madmongo1) [reply include/boost/beast/http/impl/file_body_win32.hpp]: This is how I get blamed for things I didn't say :-)

Comment (vinniefalco) [reply include/boost/beast/http/impl/file_body_win32.hpp]: Alternatively you could use VFALCO

File: include/boost/beast/core/impl/flat_buffer.hpp [hunk 2]
```diff
@@ -262,17 +260,32 @@ reserve(std::size_t n)
...
+#ifndef BOOST_NO_EXCEPTIONS
+        try
+        {
+#endif
+            p = alloc(len);
+#ifndef BOOST_NO_EXCEPTIONS
+        }
+        catch(std::exception const&)
+        {
+            // swallow this exception
```
Comment (vinniefalco) [inline include/boost/beast/core/impl/flat_buffer.hpp]: The optics on swallowing followed by an abortion are not great

Comment (madmongo1) [reply include/boost/beast/core/impl/flat_buffer.hpp]: :-D

**Pull request merged**